pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
no clue there is tons of code involved that does random stuff, its hard to say when its None while its not supposed to be like that. i can just tell calling one() without argument is broken :)
can we fix this in a own PR and i remove this from here? i don't like fixing the universe in one PR it looses the overview